### DISCUSSION TITLE
Technical Identifier for components

### DISCUSSION THREAD

**Discussion by SebastianOpriel** [IndustrialDataSpace/InformationModel#568] [Q&A]
Does a field in InformationModel exist to identify a component's technical name? E.g. something like an `ids:componentIdentifier`: 
```
"@type": "ids:BaseConnector",
"ids:version": "7.1.0",
"ids:componentIdentifier": "IDSA_DSC",
```
Due to a different usage of 
- fields (e.g. URI vs URN), 
- expected non-mandatory fields or 
- different message flows 

across different Connector implementation, a field like `componentIdenfier` would help to establish a consistency at least with known other components and its versions.

**Comment by SebastianOpriel** [comment]
@PHochmann @JohannesLipp

**Comment by JohannesLipp** [comment]
We recently added references to `ids:Described` to multiple classes (cf. [my comment here](https://github.com/International-Data-Spaces-Association/InformationModel/issues/521#issuecomment-1112880702) and the [related pull request](https://github.com/International-Data-Spaces-Association/InformationModel/pull/560), which was merged to `develop`. 

The `ids:BaseConnector` has a subClassOf chain through `ids:Connector`, `ids:InfrastructureComponent`, `ids:ManagedEntity` finally to `ids:Described`. You therefore should be able to use the fields `dct:title` and `dct:description` as you prefer (cf. the [`Described validation shape from line 43](https://github.com/International-Data-Spaces-Association/InformationModel/blob/develop/testing/shared/DescribedShape.ttl#L43)).

Please let us know if that helps, thanks!
